Source/SwiftLintFramework/Rules/Style/ColonRule.swift

```diff
@@ -2,8 +2,8 @@ import Foundation
 import SourceKittenFramework
 import SwiftSyntax
 
-public struct ColonRule: SubstitutionCorrectableRule, ConfigurationProviderRule, AutomaticTestableRule,
-                         SourceKitFreeRule {
+public struct ColonRule: SubstitutionCorrectableRule, ConfigurationProviderRule, SourceKitFreeRule,
+                         ManuallyTestedExamplesRule {
```

Comment:
How come this one changed from `AutomaticTestableRule` to `ManuallyTestedExamplesRule`?

Reply:
Good question. The `AutomaticTestableRule` conformance is not true since there is no automatically generated test. Seems like the build step checking for changes caused by `make sourcery` does not yet work. I will investigate this.

Reply:
See #4084.